CompanyEmployees.Core.Domain/Entities/Employee.cs

```diff
@@ -20,7 +20,10 @@ public class Employee
         public string? Position { get; set; }
 
         [ForeignKey(nameof(Company))]
-        public Guid CompanyId { get; set; }
+public decimal Salary { get; set; }
         public Company? Company { get; set; }
+
```

Comment:
This temporary field `salary` is redundant, does not follow C# naming conventions, and violates clean architecture principles. It should be removed before merging.

```suggestion
// Removed temporary field 'salary'
```